livekit_models.proto

```diff
@@ -73,6 +73,23 @@ message TrackInfo {
   bool disable_dtx = 8;
   // source of media
   TrackSource source = 9;
+  repeated VideoLayer layers = 10;
+}
+
+enum VideoQuality {
+  LOW = 0;
+  MEDIUM = 1;
+  HIGH = 2;
+}
+
+// provide information about available spatial layers
+message VideoLayer {
+  // for non-simulcast or SVC, this should be HIGH
```

Comment:
I am thinking SVC should report configured layers too. Not sure though if there is any event on encoder internally changing dimensions. Maybe, for now leave out comment about SVC having to use HIGH?

Reply:
yes, I meant to type non-(simulcast or SVC), but it came out wrong.. lol